There's a convoluted threading issue: in PAAudio.cpp, around line 338,
it will get caught in an infinite loop. It relies on the PortAudioInput
object being destroyed in another thread, thereby setting bRunning to
false and telling it to exit the loop, which will never happen. Luckily,
this bug is only exposed by the AudioWizard configuration stuff. So,
skip calling restartAudio() from on_qsOutputDelay_valueChanged() while
the AudioWizard is starting up

https://github.com/mumble-voip/mumble/issues/1138

Index: src/mumble/AudioWizard.cpp
--- src/mumble/AudioWizard.cpp.orig
+++ src/mumble/AudioWizard.cpp
@@ -290,7 +290,8 @@ void AudioWizard::on_qcbOutputDevice_activated(int) {
 void AudioWizard::on_qsOutputDelay_valueChanged(int v) {
 	qlOutputDelay->setText(tr("%1 ms").arg(v * 10));
 	Global::get().s.iOutputDelay = v;
-	restartAudio();
+	if (! bInit)
+		restartAudio();
 }
 
 void AudioWizard::on_qsMaxAmp_valueChanged(int v) {
